transaction/src/view/transaction_info.rs

```diff
@@ -0,0 +1,15 @@
+use crate::{Id, Transaction, TransactionPerspective, TransactionView};
+
+#[derive(Debug, Clone)]
+pub struct TransactionInfo {
+    // The height the transaction was included in a block, if known.
+    pub height: u64,
+    // The hash of the transaction.
+    pub id: Id,
+    // The transaction data itself.
+    pub transaction: Transaction,
+    // The transaction perspective, as seen by this view server.
+    pub perspective: TransactionPerspective,
+    // A precomputed transaction view of `transaction` from `perspective`, included for convenience of clients that don't have support for viewing transactions on their own.
+    pub view: TransactionView,
+}
```

Comment:
This structure is RPC-specific, so it should live in the view crate rather than the transaction crate.  And we won't be able to use it without defining a bunch of boilerplate conversions (it's not registered as a domain type, so we'd need a bunch of tryfrom/into).

Instead I think we could drop it entirely and just work with the proto types.